### PR TITLE
SOC-1251 | Bringing back sharedUrl

### PR DESCRIPTION
**Pull request by bkoval**
@rogatty @mklucsarits @kvas-damian 
https://wikia-inc.atlassian.net/browse/SOC-1251

Should be good enough for a P2 hotfix.

### CODE REVIEW

File: front/scripts/main/components/ShareFeatureComponent.ts [hunk 1]
```diff
@@ -13,6 +13,8 @@ App.ShareFeatureComponent = Em.Component.extend(App.TrackClickMixin, App.Languag
 		}
 	},
 
+	sharedUrl: Em.getWithDefault(Mercury, 'wiki.basePath', window.location.origin) + window.location.pathname,
```
Comment (rogatty) [inline front/scripts/main/components/ShareFeatureComponent.ts]: How about using `Em.computed`? @hakubo?

File: front/scripts/main/components/ShareFeatureComponent.ts [hunk 2]
```diff
@@ -13,11 +13,13 @@ App.ShareFeatureComponent = Em.Component.extend(App.TrackClickMixin, App.Languag
 		}
 	},
 
-	lineShare: Em.computed('title', function (): string {
+	sharedUrl: Em.getWithDefault(Mercury, 'wiki.basePath', window.location.origin) + window.location.pathname,
```
Comment (hakubo) [inline front/scripts/main/components/ShareFeatureComponent.ts]: will this be recomputed on 2+ page view?

**Comment by rogatty** [comment]
LGTM

**Comment by hakubo** [comment]
LgTM

File: front/scripts/main/components/ShareFeatureComponent.ts [hunk 3]
```diff
@@ -13,11 +13,16 @@ App.ShareFeatureComponent = Em.Component.extend(App.TrackClickMixin, App.Languag
 		}
 	},
 
-	lineShare: Em.computed('title', function (): string {
+	sharedUrl: Em.computed('title', function (): string {
```
Comment (mklucsarits) [inline front/scripts/main/components/ShareFeatureComponent.ts]: I just noticed "title" is required here but not used in the function.

Comment (bkoval) [inline front/scripts/main/components/ShareFeatureComponent.ts]: It needs to be here so we reload the sharedUrl each time the article changes.

Comment (mklucsarits) [inline front/scripts/main/components/ShareFeatureComponent.ts]: Oh ok. Makes sense, thanks.

Comment (bkoval) [inline front/scripts/main/components/ShareFeatureComponent.ts]: No problem! :santa:

**Pull request merged**